src/NHSD.BuyingCatalogue.Identity.Api/Services/ProfileService.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using IdentityModel;
+using IdentityServer4.Extensions;
+using IdentityServer4.Models;
+using IdentityServer4.Services;
+using NHSD.BuyingCatalogue.Identity.Api.Constants;
+using NHSD.BuyingCatalogue.Identity.Api.Infrastructure;
+using NHSD.BuyingCatalogue.Identity.Api.Models;
+using NHSD.BuyingCatalogue.Identity.Api.Repositories;
+
+namespace NHSD.BuyingCatalogue.Identity.Api.Services
+{
+    public sealed class ProfileService : IProfileService
+    {
+        private readonly IUserRepository _applicationUserRepository;
+
+        private static readonly IDictionary<string, IEnumerable<Claim>> _organisationFunctionClaims =
```

Comment:
Should this be configurable instead of hard coded?

Reply:
I think it should be in the database & it very much seems like a role attribute. Something I need to follow up with @michaelpace-bjss. These claims will relate to our authentication policies so I would want to be very careful if putting them in configuration. Usually it would require a code change to introduce a new policy along with a new claim.